src/variables.py

```diff
@@ -92,6 +100,8 @@
 
 CACHE_PATH: Final = Path(os.getenv("CACHE_PATH", "."))
 
+VAULT_PAGINATION_LIMIT: Final = int(os.getenv("VAULT_PAGINATION_LIMIT", 1_000))
+VAULT_VALIDATOR_STAGES_BATCH_SIZE: Final = int(os.getenv("VAULT_VALIDATOR_STAGES_BATCH_SIZE", 1_00))
```

Comment:
@tamtamchik you've not renamed this variable `stages` -> `statuses`
also formatting suggestion (just in case is it supposed to be 100 or 1_000?)
```suggestion
VAULT_VALIDATOR_STAGES_BATCH_SIZE: Final = int(os.getenv("VAULT_VALIDATOR_STAGES_BATCH_SIZE", 100))
```